docs/src/app/components/pages/discover-more/Showcase.js

```diff
@@ -119,6 +119,11 @@ const Showcase = () => (
           title={app.title}
           subtitle={<span>{'by '}<b>{app.author}</b></span>}
           style={styles.gridTile}
+          actionIcon={app.source &&
+            <IconButton href={app.source} target="_blank">
```

Comment:
We can't nest links. The simplest solution I can find is directly wrapping the image with the link.